aspnet/fundamentals/servers.rst

```diff
@@ -1,8 +1,274 @@
-.. include:: /../common/stub-topic.txt
+Servers
+=======
 
-|stub-icon| Servers
-===================
+By `Steve Smith`_
 
-.. include:: /../common/stub-notice.txt
+ASP.NET 5 is completely decoupled from the web server environment that hosts the application. As released, ASP.NET 5 supports IIS and IIS Express, WebListener, and Kestrel web servers, which run on a variety of platforms. Developers and third party software vendors can create their own custom servers as well within which to host their ASP.NET 5 applications.
 
-.. _issue: https://github.com/aspnet/Docs/issues/59
+In this article:
+	- `Servers and commands`_
+	- `Supported features by server`_
+	- `IIS and IIS Express`_
+	- `WebListener`_
+	- `Kestrel`_
+	- `Choosing a server`_
+	- `Custom Servers`_
+	
+`Browse or download samples on GitHub <https://github.com/aspnet/Docs/tree/master/docs/fundamentals/servers/sample>`_.
+
+Servers and commands
+--------------------
+
+ASP.NET 5 is designed to decouple web applications from the underlying web server that hosts them. Traditionally, ASP.NET applications have been windows-only (unless `hosted via mono <http://www.mono-project.com/docs/web/aspnet/>`_) and hosted on the built-in web server for windows, Internet Information Server (IIS) (or a development server, like `IIS Express <http://www.iis.net/learn/extensions/introduction-to-iis-express/iis-express-overview>`_ or earlier development web servers). While IIS is still the recommended way to host production ASP.NET applications on Windows, the cross-platform nature of ASP.NET allows it to be hosted in any number of different web servers, on multiple operating systems.
+
+ASP.NET 5 ships with support for 3 different servers:
+
+- Microsoft.AspNet.Loader.IIS (Helios)
+- Microsoft.AspNet.Server.WebListener (WebListener)
+- Microsoft.AspNet.Server.Kestrel (Kestrel)
```

Comment:
I see, but we can go live today with what it is today, and update it later as needed.